### PR TITLE
changed legend colors for imaqal graphs

### LINKED ISSUE #77: Graphs Color scheme
Duplicate of issue #69. Most of the colors in the color palettes we are using are indistinguishable when they are used on the graphs, instead we should provide color names that we can differentiate on the graphs.

### PR DESCRIPTION
**Pull request by DanielMwendwa**
Screenshot from 2019-10-11 17-06-48
Fixes: #77

### CODE REVIEW

File: public/media_engagement/imaqal_media_engagement.js [hunk 1]
```diff
@@ -224,7 +224,7 @@ const update = (data) => {
     const total_failed_path = total_failed_sms_graph.append('path');
 
     // custom color scheme
-    color_scheme = ["#FF0000", "#f1a201", "#3cb44b", "#181eb4", "#ffe119", "#9A6324", "#fe0991", "#911eb4"]
+    color_scheme = ["#e6194B", "#f58231", "#3cb44b", "#CCCC00", "#4363d8", "#800000", "#f032e6", "#911eb4"]
```
Comment (IsaackMwenda) [inline public/media_engagement/imaqal_media_engagement.js:227]: please upload a screenshot of  the new changes and request re-review

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

**Comment by DanielMwendwa** [comment]
legends

**Review by IsaackMwenda — APPROVED** [review]
Thanks @DanielMwendwa

**Review by lukechurch — APPROVED** [review]
Thanks @DanielMwendwa

**Pull request merged**